drivers/growatt/driver.js

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019: Rob, https://github.com/pro-sumer
+// License: GNU GPLv3, https://www.gnu.org/licenses/gpl.txt
+
+'use strict'
+
+const Homey = require('homey')
+const api = require('./api')
+
+class Growatt extends Homey.Driver {
+  onPair (socket) {
+    let username
+    let password
+    let devices = []
+
+    socket.on('login', async (data, callback) => {
+      username = data.username
+      password = data.password
+      try {
+        const loginResponse = await api.login(username, password)
+        const loginData = await loginResponse.json()
+        if (loginData.back.success) {
+          for (let inverter of loginData.back.data) {
+            devices.push({
+              name: inverter.plantName,
```

Comment:
Is this one always filled? I'm not sure if Growatt forces users to give their plant a name, and I'm not entirely sure what the Athom SDK does when the name is an empty string or even null/undefined.

Reply:
Not sure whether this can happen, but applied defensive programming by adding `|| "Inverter"`